src/components/Date/Date.stories.jsx

```diff
@@ -61,3 +57,9 @@ Error.args = {
     },
   },
 };
+
+export const AriaDescribedby = Template.bind({});
+AriaDescribedby.args = {
+  ...defaultArgs,
+  ariaDescribedby: 'some-id',
```

Comment:
Do we have `some-id` on the page?

Reply:
No, not in the stories. I wasn't sure how to add it to the stories page.